artiq/frontend/artiq_coremgmt.py

```diff
@@ -82,10 +82,15 @@ def get_argparser():
 
     subparsers.add_parser("erase", help="fully erase core device config")
 
+    # flash
+    t_flash = tools.add_parser("flash",
+                        help="ARTIQ reboot tool though internet")
+
+    t_flash.add_argument("action", metavar="ACTION", nargs="*",
+                        default=[],
+                        help="actions to perform, choose from: start")
+    
     # booting
-    t_boot = tools.add_parser("reboot",
-                              help="reboot the currently running firmware")
-
     t_hotswap = tools.add_parser("hotswap",
```

Comment:
Remove this one, it doesn't work correctly. Though that can go into a separate PR.